### ISSUE TITLE
"PROPERTY_HINT_DICTIONARY_TYPE" not declared in current scope

### ISSUE THREAD

**Issue by RowanSkie** [chairfull/GodotRichTextLabel2#14]
```
 res://addons/richtext2/richer_text_label.gd:1110 - Parse Error: Identifier "PROPERTY_HINT_DICTIONARY_TYPE" not declared in the current scope.
  res://addons/richtext2/rich_text_animation.gd:2 - Parse Error: Could not resolve class "RicherTextLabel".
  res://addons/richtext2/text_effects/rich_text_effect_base.gd:-1 - Compile Error: 
  res://addons/richtext2/richer_text_label.gd:1110 - Parse Error: Identifier "PROPERTY_HINT_DICTIONARY_TYPE" not declared in the current scope.
  res://addons/richtext2/text_effects/anims/rte_back.gd:-1 - Compile Error: 
  modules/gdscript/gdscript.cpp:2936 - Failed to load script "res://addons/richtext2/text_effects/anims/rte_back.gd" with error "Compilation failed". (User)
  res://addons/richtext2/richer_text_label.gd:1110 - Parse Error: Identifier "PROPERTY_HINT_DICTIONARY_TYPE" not declared in the current scope.
  res://addons/richtext2/rich_text_animation.gd:-1 - Compile Error: 
  res://addons/richtext2/text_effects/anims/rte_console.gd:-1 - Compile Error: 
  modules/gdscript/gdscript.cpp:2936 - Failed to load script "res://addons/richtext2/text_effects/anims/rte_console.gd" with error "Compilation failed". (User)
  res://addons/richtext2/richer_text_label.gd:1110 - Parse Error: Identifier "PROPERTY_HINT_DICTIONARY_TYPE" not declared in the current scope.
  res://addons/richtext2/text_effects/effects/rte_jump.gd:-1 - Compile Error: 
  modules/gdscript/gdscript.cpp:2936 - Failed to load script "res://addons/richtext2/text_effects/effects/rte_jump.gd" with error "Compilation failed". (User)
  res://addons/richtext2/richer_text_label.gd:1110 - Parse Error: Identifier "PROPERTY_HINT_DICTIONARY_TYPE" not declared in the current scope.
  res://addons/richtext2/text_effects/effects/rte_sin.gd:-1 - Compile Error: 
  modules/gdscript/gdscript.cpp:2936 - Failed to load script "res://addons/richtext2/text_effects/effects/rte_sin.gd" with error "Compilation failed". (User)
  ```
  that is normal, right? Or is PROPERTY_HINT_DICTIONARY_TYPE for 4.4-preview or something?

**Comment by aiaimimi0920** [comment]
The author has fixed this issue in 1.6.1. Please download the latest version
@RowanSkie 
https://github.com/chairfull/GodotRichTextLabel2/issues/6

**Comment by stwalsh4118** [comment]
Still getting this issue in 1.7.1 version
Offending line of code:
https://github.com/chairfull/GodotRichTextLabel2/blob/27dc3aaf521ae8ec178e92f2e7e71a0b0ac35e32/addons/richtext2/richer_text_label.gd#L1110C1-L1110C102

**Comment by RowanSkie** [comment]
> The author has fixed this issue in 1.6.1. Please download the latest version @RowanSkie #6

I'm downloading via the on-app AssetLib tab, and like @stwalsh4118 said, it's still on the 1.7.1 version.

**Comment by aiaimimi0920** [comment]
The author should have introduced this error again in the latest update,
You can modify it to: 
```
_prop(props, &"context_state", TYPE_DICTIONARY)
```
It should be able to run

By the way, the author is probably developing this plugin using version 4.4dev, so if you want to use it smoothly, it's best to update the version (although I also hope he is developing plugins for version 4.3)
